File: Objetivo 3/HerancaSimplesExemploPHP/model/ContaCorrente.php
<?php

include_once "Conta.php";

/**
 * Description of ContaCorrente
 *
 * @author vagner
 */
class ContaCorrente extends Conta {
    
    public function saca($valor) {
        if($this->saldo >= $valor){ 
            $this->saldo = $valor;
        }
    }
  
}

File: Objetivo 3/HerancaSimplesExemploJava/src/model/ContaCorrente.java
package model;

/**
 *
 * @author vagner
 */
public class ContaCorrente extends Conta{

    @Override
    public void saca(double valor) { //sobreescreve o comportamento da classe Conta
        if(saldo >= valor){ //(getSaldo() >= valor)
            saldo -= valor; //saca(valor);
        }
        
    }
    
}
